docs/config/generate-api-certificates.rst

```diff
@@ -1,16 +1,16 @@
 .. _generate-api:
 
-==================================
-Generating an API User Certificate
-==================================
+====================================
+Generating an API Client Certificate
+====================================
 
-To send instructions to the control service, whether it is via the API directly, or the CLI, or by any other method, you will need to follow the instructions below to generate an API user certificate:
+To send instructions to the control service, whether it is via the API directly, or the CLI, or by any other method, you will need to follow the instructions below to generate an API client certificate:
 
-#. Generate an API user certificate:
+#. Generate an API client certificate:
 
    Run the following command from the directory which contains the certificate authority files generated when you first installed the cluster. For more information, see :ref:`authentication`.
 
-   Replace ``<username>`` with a unique username for an API user.
+   Replace ``<username>`` with a unique username for an API client.
```

Comment:
"unique identifier" maybe, suggesting username as an example?

Also, probably should change `username` to `client_name` in  the examples.